client/state/action-types.js

```diff
@@ -262,6 +262,9 @@ export const GUIDED_TRANSFER_STATUS_REQUEST = 'GUIDED_TRANSFER_STATUS_REQUEST';
 export const GUIDED_TRANSFER_STATUS_REQUEST_FAILURE = 'GUIDED_TRANSFER_STATUS_REQUEST_FAILURE';
 export const GUIDED_TRANSFER_STATUS_REQUEST_SUCCESS = 'GUIDED_TRANSFER_STATUS_REQUEST_SUCCESS';
 export const GUTENBERG_OPT_IN_DIALOG_IS_SHOWING = 'GUTENBERG_OPT_IN_DIALOG_IS_SHOWING';
+export const GUTENBERG_CREATE_POST_DRAFT = 'GUTENBERG_CREATE_POST_DRAFT';
+export const GUTENBERG_SITE_POST_REQUEST = 'GUTENBERG_SITE_POST_REQUEST';
+export const GUTENBERG_SITE_POST_RECEIVE = 'GUTENBERG_SITE_POST_RECEIVE';
```

Comment:
In terms of naming should we be prefixing with Gutenberg? These are core v2 schema posts right?

I'll also recommend a more descriptive name that would allow some of these actions to make sense out of the context of any server action:
`*POST_ADD` vs `*_SITE_POST_RECEIVE`
`*POST_DRAFT_ADD` vs `* _CREATE_POST_DRAFT` (it sounds weird but actions should be postfixed).

Reply:
> In terms of naming

the core project has been starting to eschew the name `Gutenberg` in code review. I hope this makes your day more interesting. they are moving to internalize the name "editor" and the like.

I ran into this while working on the parser stuff - renames from things like `Gutenberg_Block` to `WP_Block`

just wanted to throw that out there 😬